kivymd/uix/refreshlayout/refreshlayout.py

```diff
@@ -175,8 +177,26 @@ class MDScrollViewRefreshLayout(MDScrollView):
     and defaults to `None`.
     """
 
+    spinner_color = ColorProperty()
+    """
+    Color of spinner in (r, g, b, a) or string format.
+
+    :attr:`spinner_color` is a :class:`~kivy.properties.ColorProperty`
+    and defaults to `[1, 1, 1, 1]`.
+    """
+
+    circle_color = ColorProperty(None)
+    """
+    Color of ellipse around the spinner in (r, g, b, a) or string format.
+
+    :attr:`circle_color` is a :class:`~kivy.properties.ColorProperty`
+    and defaults to `[1, 1, 1, 1]`.
```

Comment:
and defaults to `None`.

Reply:
The same, there are no need to set `None` by default and later reassign `[1, 1, 1, 1]`